move/axelar_gateway/sources/events.move

```diff
@@ -48,11 +48,7 @@ public struct MessageExecuted has copy, drop {
 // -----------------
 // Package Functions
 // -----------------
-public(package) fun signers_rotated(
-    epoch: u64,
-    signers_hash: Bytes32,
-    signers: WeightedSigners,
-) {
+public(package) fun signers_rotated(epoch: u64, signers_hash: Bytes32, signers: WeightedSigners) {
```

Comment:
after adding the move formatter gh action it'll be easier to separate formatting changes from main ones